### PR TITLE
Added diagram showing the commercial products, their relations and dependencies.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Added a new pages and diagram showing the commercial products, their relations and dependencies.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [X] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

All commercial products.

## Deadline (if relevant)

Can be reviewed at any time.  Please though wait for any comments from @BoletteKern and @mattbrailsford  too.

### CODE REVIEW

File: marketplace-and-integrations/README.md
Comment (alina-tincas) [inline marketplace-and-integrations/README.md:35]: AFAIK [.net frameworks ](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework) are not same as what v10+ use it is just .net. I also added a link to it.
```suggestion
We aim to make all commercial products available as new major versions in alignment with the releases of Umbraco CMS. A new major version of Umbraco, running on the latest version of [.NET](https://dotnet.microsoft.com/en-us/download/dotnet), is expected every six months. At or around the same time, compatible Forms, Deploy, Workflow and Commerce releases will also be made available, along with [the additional, dependent packages](./commercial-product-dependencies.md).
```

**Review by alina-tincas — APPROVED** [review]
Hi @AndyButland thank you for the PR 🙌 

That is a nice diagram 😁 I have just one suggestion, otherwise, all looks good!

I will await approval from @BoletteKern and @mattbrailsford before merging this in. 👍

**Comment by BoletteKern** [comment]
I think it looks great!

**Comment by mattbrailsford** [comment]
Looks good to me too 👍

**Comment by alina-tincas** [comment]
Thank you for approving @BoletteKern and @mattbrailsford 🙌 

Merging it in 💪

**Pull request merged**
